src/xdocs/config_coding.xml

```diff
@@ -762,6 +784,16 @@ String nullString = null;
         <source>
 &lt;module name=&quot;ExplicitInitialization&quot;/&gt;
         </source>
+
+        <p>
+          To configure the check so that it only checks
+          for objects that explicitly initialize to null:
+        </p>
+        <source>
+&lt;module name=&quot;ExplicitInitialization&quot;&gt;
+    &lt;property name=&quot;onlyObjectReferences&quot; value=&quot;true&quot;/&gt;
+&lt;/module&gt;
+        </source>
```

Comment:
Please add Java example below that will show which will be violations and which won't when this config is used.